new-dti-website/src/app/apply/apply-timeline.tsx

```diff
@@ -0,0 +1,185 @@
+
+/* eslint-disable no-param-reassign */
```

Comment:
:(

Reply:
https://stackoverflow.com/questions/35637770/how-to-avoid-no-param-reassign-when-setting-a-property-on-a-dom-object